tensorboard/plugins/audio/summary_v2.py

```diff
@@ -65,6 +66,9 @@ def audio(
         default, so if you want "wav" in particular, set this explicitly.
       description: Optional long-form description for this summary, as a
         constant `str`. Markdown is supported. Defaults to empty.
+      lengths: Optional lengths `Tensor`, shaped `[k]`, where each entry
```

Comment:
Could you slightly expand the documentation here so it's a little more obvious why one would want to use this parameter and exactly what it does?  e.g. something like the following (added wording in italics)

> Optional lengths `Tensor`, shaped `[k]`, where each entry indicates the length of the k'th example in frames. _Useful when `data` should be interpreted as ragged rather than fixed-length; each example will be truncated accordingly prior to audio encoding._ k may be statically unknown (i.e., `None`).

I would suggest adding a note to `data` as well, something like:

> A `Tensor` representing audio data with shape `[k, t, c]`, where `k` is the number of audio clips, `t` is the number of frames, and `c` is the number of channels. _To pass batches of data where the frame count is variable, use the `lengths` parameter._ Elements should be floating-point values in `[-1.0, 1.0]`. Any of the dimensions may be statically unknown (i.e., `None`).

Reply:
Good suggestions! Done.